File: app/src/main/java/com/saitejajanjirala/imdbclone/models/BaseResponse.java
package com.saitejajanjirala.imdbclone.models;

import com.google.gson.annotations.SerializedName;

public class BaseResponse {
    @SerializedName("status_message")
    public String statusMesssage;

    @SerializedName("status_code")
    public int statusCode;
}

File: app/src/main/java/com/saitejajanjirala/imdbclonekotlin/models/BaseResponse.kt
package com.saitejajanjirala.imdbclonekotlin.models

import com.google.gson.annotations.SerializedName

open class BaseResponse(
    @SerializedName("status_message")
    var statusMesssage: String? = null,
    @SerializedName("status_code")
    var statusCode: Int = 0
)